docs/dev.rst

```diff
@@ -1,20 +1,540 @@
+************************************************
+
+When the ``update_dependencies`` task runs, it retrieves a list of all managed packages in the target org, and creates a list of the installed packages and their version numbers.
+
+    Example: ``npe01 version 3.6``
+    
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - namespace: npe01
+              version: 3.6
+    
+..    
+    
+    Depending on whether or not the package with namespace ``npe01`` is installed, the ``update_dependencies`` task runs these steps. 
+
+    * If ``npe01`` is not installed, ``npe01 version 3.6`` is installed.
+    * If the org already has ``npe01 version 3.6`` installed, no changes take place.
+    * If the org has an older version installed, it's upgraded to ``version 3.6``.
+    * If the org has a newer version or a beta version installed, it's uninstalled and ``version 3.6`` is installed.
+
+
+Hierarchical Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Managed package dependencies can handle a hierarchy of dependencies between packages.
+
+    Example: Salesforce.org's Nonprofit Success Pack (NPSP), an extension of five other managed packages, one of which (Households) is an extension of another (Contacts & Organizations).
+
+    These dependencies are listed under the ``project`` section of the ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - namespace: npo02
+              version: 3.8
+              dependencies:
+                  - namespace: npe01
+                    version: 3.6
+            - namespace: npe03
+              version: 3.9
+            - namespace: npe4
+              version: 3.5
+            - namespace: npe5
+              version: 3.5
+
+..
+
+    The project requires ``npo02 version 3.8``, which itself requires ``npe01 version 3.6``. By specifying the dependency hierarchy, the ``update_dependencies`` task is capable of uninstalling and upgrading packages intelligently.
+
+    So if the target org currently has ``npe01 version 3.7``, ``npe01`` needs to be uninstalled to downgrade to ``3.6``. However, ``npo02`` requires ``npe01``, so uninstalling ``npe01`` also requires uninstalling ``npo02``. (In this scenario ``npe03``, ``npe04``, and ``npe05`` do not have to be uninstalled to uninstall ``npe01``.)
+
+
+Unmanaged Metadata Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify unmanaged metadata to be deployed by specifying a ``zip_url`` and, optionally, ``subfolder``, ``namespace_inject``, ``namespace_strip``, and ``unmanaged`` under the ``project__dependencies`` section of the cumulusci.yml file.
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - zip_url: https://SOME_HOST/metadata.zip
+
+When the ``update_dependencies`` task runs, it downloads the zip file and deploys it via the Metadata API's ``deploy`` method. The zip file must contain valid metadata for use with a deploy, including a ``package.xml`` file in the root.
```

Comment:
Metadata API's ``deploy`` method

Deploy should be in backquotes, correct?

Reply:
Probably, but even better, just delete after `Metadata API` to the end of the sentence.